### DISCUSSION TITLE
Idées liées à l'enquête de satisfaction

### DISCUSSION THREAD

**Discussion by gauthierandre** [betagouv/ma-cantine#1696] [Idées]
Voici quelques pistes d'amélioration qui me viennent spontanément (et vivement suggérées par les avis des utilisateurs) après l'écriture de ce document : 
* création d'outils de formation / visibilisation du contenu vidéo déjà présent sur le site (ex : intégrer les vidéos tutos à chaque étape plutôt que dans le Gitbook) et disponible à tous ! 
* rédiger un tutoriel général de la plateforme (en cours grâce à la campagne de remontée de données)
* créer des profils avec des droits de lecture seulement et non d'édition ; 
* un des points forts de la plateforme : son caractère "œcuménique" : il faut donc continuer de proposer des documents différents, de valoriser les initiatives des acteurs ;  
* dans la lignée précédente : est-ce qu'on pourrait intégrer des éléments du gitbook en menu déroulant ? Ex : page recette / page gaspi ou intéger les éléments sur les cartes correspondantes ;

**Comment by valeriemerle** [comment]
Pleins de bonnes idées ces utilisateurs...
